src/libs/actions/Task.ts

```diff
@@ -105,12 +118,12 @@ function createTaskAndNavigate(parentReportID, title, description, assigneeEmail
         {
             onyxMethod: Onyx.METHOD.SET,
             key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${optimisticTaskReport.reportID}`,
-            value: {[optimisticTaskCreatedAction.reportActionID]: optimisticTaskCreatedAction},
+            value: {[optimisticTaskCreatedAction.reportActionID ?? '']: optimisticTaskCreatedAction as OnyxTypes.ReportAction},
```

Comment:
You have many places where you add nullish coalescing operator to `reportActionID` value, cause it's typed as optional.At the same time if you take a look at how it's [build](https://github.com/Expensify/App/blob/main/src/libs/ReportUtils.ts#L3156) 'reportActionID' always will be there, maybe we can update `OptimisticCreatedReportAction` type to take it into consideration? But it's up to you